hw23/src/main/java/org/example/builder/CarBuilder.java

```diff
@@ -0,0 +1,28 @@
+package org.example.builder;
+
+public class CarBuilder {
```

Comment:
wrong implementation of builder pattern, use static selfcalling method that returns "this" object.
you should be able to use builder in the following way:

Car car = CarBuilder.shell("shell").electric("electric").wheel("some wheel").build().